### DISCUSSION TITLE
laravel vue 3 pagination not shown in template

### DISCUSSION THREAD

**Discussion by umerghori786** [gilbitron/laravel-vue-pagination#174] [Help & support]
i am facing an issue i installed Laravel Vue Pagination today but its view not comes on template here it is my code 
```
<template>
  <div>
    <ul>
      <li v-for="post in laravelData.data" :key="post.id">{{ post.title }}</li>
    </ul>

    <Bootstrap4Pagination
        :data="laravelData"
        @pagination-change-page="getResults"
    />
  </div>
</template>

<script setup>
import { ref } from 'vue';
import { Bootstrap4Pagination } from 'laravel-vue-pagination';

const laravelData = ref({});

const getResults = async (page = 1) => {
    const response = await fetch(`/api/products?page=${page}`);
    laravelData.value = await response.json();
}

getResults();
</script>
```
here it show product response return 
![image](https://github.com/gilbitron/laravel-vue-pagination/assets/65702036/01e4f5f4-d522-4a26-a92f-f78cd4a92f1b)
 here i see nothing UI of pagination
![image](https://github.com/gilbitron/laravel-vue-pagination/assets/65702036/c799904f-4bcc-4e71-8353-314297a31dd7)

thanks

**Comment by umerghori786** [comment]
can you tell suggestion

**Reply by luoxiangxu** [reply]
i have the same problem

**Reply by luoxiangxu** [reply]
do you have the solution?

**Comment by repeater09** [comment]
I dont know if I will help you guys since it's been a while but what worked for me was creating a resource/collection for the data I was using. 

When I did that it properly laid out the data object along with the links object that this looks for.
